src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPresenter.cs

```diff
@@ -10,8 +14,19 @@ public FlyoutPresenter()
 
 		protected override void OnPointerPressed(PointerRoutedEventArgs args)
 		{
-			// All pointer-related should be "eaten" to prevent closing
-			// the flyout when a tap is done in its content
+			if (this.GetTemplateRoot() is FrameworkElement root && this.Parent is FlyoutBasePopupPanel panel)
+			{
+				// allow flyout to be closed by clicking outside its content
+				var pointer = args.GetCurrentPoint(this);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'args' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5036884)